src/styles/theme/index.ts

```diff
@@ -114,6 +114,8 @@ export const common = {
         six: '#497BFF',
         seven: '#FF7549',
         eight: '#EB3AA4',
+        nine: '1E3B93',
+        ten: '4F12D1',
```

Comment:
I'm surprised these colors are showing up? You updated it here but didn't update the Tag component. 

Nothing to fix here though. I'm going to merge a PR with 10 new colors today.

Reply:
I have rebased this branch with base branch and applied your changes